content/en/tracing/trace_collection/library_config/dotnet-core.md

```diff
@@ -166,6 +166,11 @@ Accepts a comma-separated list of key-value pairs of case-insensitive header key
 **Example**: `CASE-insensitive-Header:my-tag-name,User-ID:userId,My-Header-And-Tag-Name`<br>
 Added in version 1.18.3. Response header support and entries without tag names added in version 1.26.0.
 
+`DD_TRACE_CLIENT_IP_ENABLED`
+**Default**: `false`<br>
+Enables client IP collection from relevant IP headers.<br>
+Added in version `2.19.0`.<br>
```

Comment:
```suggestion
`DD_TRACE_CLIENT_IP_ENABLED`
: Enables client IP collection from relevant IP headers.<br>
Added in version `2.19.0`.<br>
**Default**: `false`<br>
```

Reply:
Thanks, changes accepted in the last commit.